appinfo/application.php

```diff
@@ -36,7 +37,7 @@ public function __construct(array $urlParams=array()){
 			['locking' => false]);
 
 
-		$container->registerService('HttpBindController', function($c){
+		$container->registerService('HttpBindController', function(IContainer $c) {
```

Comment:
That's not necessary to do, it just allows simple type hinting. Otherwise, my IDE doesn't know what `$c` is, now it properly detects its type and can type hint the functions.